oem/firstvoices/common/build_keyboards.sh

```diff
@@ -103,4 +122,40 @@ if [ $DO_BUILD = true ] || [ $DO_COPY = true ]; then
   popd
 fi
 
+if [ $DO_DOWNLOAD = true ]; then
+  echo "Downloading keyboards from downloads.keyman.com"
+
+  cp ../keyboards.csv "$KEYBOARDS_CSV_TARGET"
+
+  SCRIPT_ROOT=`pwd`
+  URL_DOWNLOAD=https://downloads.keyman.com
+  URL_API_VERSION=${URL_DOWNLOAD}/api/keyboard/
+
+  case "${OSTYPE}" in
```

Comment:
It's possible to build the Android app on macOS. So this switch is inadequate. We probably need to pass an additional parameter (`-kmp`) into the script which we can eliminate once the iOS code consumes .kmp.

Reply:
I've changed iOS to use a parameter `-download-js-keyboards`.